com.unity.render-pipelines.high-definition/Runtime/Lighting/Shadow/ScreenSpaceShadowManagerDirectional.cs

```diff
@@ -271,11 +263,12 @@ SSShadowDirectionalParameters PrepareSSShadowDirectionalParameters()
             return sssdParams;
         }
 
-        static void ExecuteSSShadowDirectional(CommandBuffer cmd, SSShadowDirectionalParameters sssdParams, RTHandle textureArray)
+        static void ExecuteSSShadowDirectional(CommandBuffer cmd, SSShadowDirectionalParameters sssdParams, RTHandle normalBuffer, RTHandle textureArray)
         {
             // If it is screen space but not ray traced, then we can rely on the shadow map
             // WARNING: This pattern only works because we can only have one directional and the directional shadow is evaluated first.
             CoreUtils.SetRenderTarget(cmd, textureArray, depthSlice: sssdParams.depthSlice);
+            cmd.SetGlobalTexture(HDShaderIDs._NormalBufferTexture, normalBuffer);
```

Comment:
Using SetGlobal is dangerous in the context of RenderGraph since it breaks the contract about lifetime of textures.
Please use a MaterialPropertyBlock along with the material to set this parameter.